verify/verify.go

```diff
@@ -759,7 +768,7 @@ func verifyPCKCertificationChain(options *Options) error {
 	return checkCertificateExpiration(chain, options)
 }
 
-func x509Options(trustedRoots *x509.CertPool, intermediateCert *x509.Certificate) x509.VerifyOptions {
+func x509Options(trustedRoots *x509.CertPool, intermediateCert *x509.Certificate, options *Options) x509.VerifyOptions {
```

Comment:
This should just take in the time instead of the entire options struct. This aligns with the rest of the params which are used to return the VerifyOptions

Reply:
Done